docs/UserGuide.md

```diff
@@ -469,136 +461,121 @@ You can use the delete command to delete the company from the list of companies
 
 **Example:**
 
-If you've completed the quick tour of LinkMeIn, you've likely deleted a few companies.
-If you have not done so yet, you can refer to the [Quick Tour](#4-deleting-a-company) for guidance on deleting companies.
-Here are some examples of `delete` command that you can try:
+If you've completed the Quick Tour of LinkMeIn, you've likely deleted a few companies. If not, you can refer to the [Quick Tour Section](#4-deleting-a-company) for guidance on deleting companies.
+
+Here are some other examples of `delete` commands that you can try:
 - `delete 1`
 - `delete 2`
 
 **Possible Error:**
 
-If you enter an [**invalid index**](#5-what-is-an-invalid-index),
-you will see an error message in the command box. Don’t worry, just edit your index accordingly and
-try the command again!
-An example of an error message is shown below.
+If you have entered an [**invalid index**](#5-what-is-an-invalid-index),
+you will see an error message in the Message Box as shown below. Don’t worry, just edit your index accordingly and try the command again!
 
 ![img.png](images/delete-command/DeleteCommandError.png)
 
 [&uarr; Back to Table of Contents](#table-of-contents)
 
 ### Adding Remarks to a Company: `remark`
 
-_After adding a company, you wish to note down more information such as the job description, pre-requisites and mental notes for yourself! Just use the `remark` command to input these additional details._
+_After adding a company, you wish to note down more information such as the job description, pre-requisites and mental notes for yourself! Just use the `remark` command to input these additional details!_
 
 **Format:** `remark INDEX re/REMARK`
 
-> `INDEX`: Adds remarks to the company at the INDEX number shown in the displayed company list.  
-> `re/REMARK` : Type prefix `re/` followed by the remarks you want to add. 
-
-<div markdown="block" class="alert alert-info">
-**:information_source: Note:**<br>
-* Remarks cannot be empty. If you wish to remove existing remarks, use the  [`unremark` command](#deleting-remarks-of-a-company) instead.
-</div>
```

Comment:
I think this makes sense tho? Maybe can be a tip instead? Under the possible error section

Reply:
okay, will add that in